src/Command/BakeCommand.php

```diff
@@ -196,4 +196,32 @@ protected function parseFile(string $path): ?ParsedFile
 
         return null;
     }
+
+    /**
+     * Write file contents out to path and prompt user with options with file exists.
+     *
+     * @param \Cake\Console\ConsoleIo $io Console io
+     * @param string $path The path to create the file at
+     * @param string $contents The contents to put into the file
+     * @param bool $forceOverwrite Whether the file should be overwritten without prompting the user
+     * @param bool $skipSame Skip writing output if the contents match existing file
+     * @return bool True if successful, false otherwise
+     * @throws \Cake\Console\Exception\StopException When `q` is given as an answer
+     *   to whether a file should be overwritten.
+     */
+    protected function writeFile(
+        ConsoleIo $io,
+        string $path,
+        string $contents,
+        bool $forceOverwrite = false,
+        bool $skipSame = true
```

Comment:
```suggestion
        bool $skipIfSame = true
```
Sounds better

Reply:
I wasn't sure, but seeing that, I wonder if `skipIfUnchanged` is clearer. I used that name in the tests.

Reply:
`skipIfUnchanged` 👍